detach.py

```diff
@@ -358,8 +358,8 @@ def run(maildir, smtp_conn, exclude_seen, dir_pattern, learn_spam, learn_ham):
         smtp_host = cfg.get("smtp", "host", fallback="localhost")
         smtp_port = cfg.getint("smtp", "port", fallback=25)
         pattern = cfg.get("detach", "pattern")
-        dir_pattern = pattern+cfg.get("detach", "pattern")
-	url_pattern = pattern+cfg.get("detach", "pattern")
+        dir_pattern = pattern+cfg.get("detach", "dir")
+	url_pattern = pattern+cfg.get("detach", "url")
```

Comment:
I think the sub strings should be switched - now it puts the folder structure before the url part.